File: ConsoleApp1/Data/conexao.cs
using Npgsql;

namespace ConsoleApp1
{
    public class Conexao
    {
        private static string connectionString =
            "Host=localhost;" +
            "Port=5432;" +
            "Username=postgres;" +
            "Password=ADMIN;" +
            "Database=postgres";

        public static NpgsqlConnection ObterConexao()
        {
            return new NpgsqlConnection(connectionString);
        }
    }
}

File: conexao.vb
Imports Npgsql

Public Class Conexao

    Private Shared connectionString As String =
        "Host=localhost;Port=5432;Username=postgres;Password=ADMIN;Database=postgres"

    Public Shared Function ObterConexao() As NpgsqlConnection
        Return New NpgsqlConnection(connectionString)
    End Function

End Class
